plugins/modules/events_and_notifications_workflow_manager.py

```diff
@@ -1048,9 +1048,8 @@ def collect_snmp_playbook_params(self, snmp_details):
             'snmpVersion': snmp_details.get('snmp_version')
         }
         server_address = snmp_details.get('server_address')
-        pattern = re.compile(r'^[A-Za-z0-9]([A-Za-z0-9.:-]*[A-Za-z0-9])?$')
 
-        if server_address and not re.match(pattern, server_address):
+        if server_address and not self.is_valid_server_address(server_address):
             self.status = "failed"
             self.msg = "Invalid server adderess '{0}' given in the playbook for configuring SNMP destination".format(server_address)
```

Comment:
Can you please check the spelling.
Invalid server **address**

Reply:
updated.